pybossa/view/projects.py

```diff
@@ -2992,21 +2991,32 @@ def answerfieldsconfig(short_name):
 
     if request.method == 'POST':
         try:
-            project.info['answer_fields'] = json.loads(request.data)
+            body = json.loads(request.data) or {}
+            answerFieldsConfig = body.get('answerFieldsConfig') or {}
+            consensusConfig = body.get('consensusConfig') or {}
+            if answerFieldsConfig:
```

Comment:
Looking at this now, it seems that prior to this change, there was

```
project.info['answer_fields'] = json.loads(request.data)
```

which didn't care if `answerFieldsConfig` was empty (i.e. `project.info['answer_fields']` could be "blanked out").  I'm not sure if this is needed or if `answerFieldsConfig` can be removed in another way.

Reply:
true I fix it in latest commit